Twitch.Net/TwitchApi.Streams.cs

```diff
@@ -74,11 +81,56 @@ public async Task<HelixPaginatedResponse<HelixStream>> GetStreams(string[] userI
             {
                 parameters.Add(new KeyValuePair<string, string>("before", before));
             }
-
+		
+	    if (string.IsNullOrEmpty(before) == false)
+            {
+                parameters.Add(new KeyValuePair<string, string>("before", before));
+            }
+		
             var responseStream = await httpClient.GetAsync(_getStreamsEndpoint, parameters);
 
             return await JsonSerializer.DeserializeAsync<HelixPaginatedResponse<HelixStream>>(responseStream);
         }
+        
+        /// <summary>
+        /// Get streams for specific usernames
```

Comment:
```suggestion
        /// Get streams for specific user logins
```
Ich denke, dass das passender ist, da es konsistenter mit der Benennung von der Helix API ist. Die Dokumentation bei der Überladung mit den `userIds` sollte dementsprechend auch angepasst werden.